<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="BUTTON_PLAY_PAUSE">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>368</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/audio/UI/pause_200.png);
	border: none;
}

QPushButton:pressed {
	border-image: url(:/audio/UI/pause_pressed_200.png);
	border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_ICON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_PLAY_DURATION">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>280</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QSlider" name="SLIDER_PLAY_POSITION">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>761</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
    border: none;
    height: 30px;
    margin-top: -5px;
    margin-bottom: -5px;
	margin-left: -15px;
	margin-right: -15px;
}

QSlider::handle:horizontal {
	background: black;
	border: 1px solid #5c5c5c;
	width: 30px;
	height: 30px;
    border-radius: 15px;
}

QSlider::add-page:horizontal {
/*
    margin-top: -5px;
    margin-bottom: -5px;
*/
    background: gray;
}

QSlider::sub-page:horizontal {
/*    
	margin-top: -5px;
    margin-bottom: -5px;
*/
    background: black;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_PLAY_POSITION">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_PLAY_START">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>368</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/audio/UI/play_200.png);
	border: none;
}

QPushButton:pressed {
	border-image: url(:/audio/UI/play_pressed_200.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_PLAY_PREV">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>368</y>
     <width>255</width>
     <height>150</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/audio/UI/previous_150.png);
	border: none;
}

QPushButton:pressed {
	border-image: url(:/audio/UI/previous_pressed_150.png);
	border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_PLAY_NEXT">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>368</y>
     <width>225</width>
     <height>150</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/audio/UI/next_150.png);
	border: none;
}

QPushButton:pressed {
	border-image: url(:/audio/UI/next_pressed_150.png);
	border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_MDEDIA_ETC">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>681</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_MDEDIA_TITLE">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_REPEAT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-image: url(:/audio/UI/controls-repeat-none.png);
	border: none;
}

QPushButton:pressed {
	
}

QPushButton:disabled {
	background-image: url(:/audio/UI/controls-repeat-none.png);
	border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_SHUFFLE">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-image: url(:/audio/UI/controls-shuffle-off.png);
	border: none;
}

QPushButton:pressed {
	
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
